iset.mm

```diff
@@ -155301,26 +155342,28 @@ C_ if ( A. k e. suc J ( Q ` ( i e. _om
       = ( G ` ( i e. _om |-> if ( i e. n , 1o , (/) ) ) ) ) $.
     $( Equality of two functions on ` NN+oo ` which agree at every integer and
        at the point at infinity.  From an online post by Martin Escardo.
-       (Contributed by Jim Kingdon, 4-Aug-2023.) $)
+       Remark/TODO: the last two hypotheses can be grouped into one,
```

Comment:
Hmm, the notation that most people use treats the point at infinity quite separately from an element of ` NN+oo ` corresponding to an integer (I think the latter was underlined integers or something in at least one paper, anyway some small adornment around an integer, as opposed to the rather more elaborate combination of maps-to and ` if ` notation we have here).

Not sure that really tells us what to do, but something to think about in terms of whether we really want to combine those two cases.

Reply:
You're right, maybe I'm a bit too assertive; I'll change "Remark/TODO" to "Remark".

Escardo's paper uses $\underbar{n}$ for the sequence with $n$ ones followed by zeroes, that is, the image of $n$ under the canonical embedding. He then defines $\underbar{\mathbb{N}}$ as the set of $\underbar{n}$'s.
I think it makes sense to extend this to $\underbar{+\infty}$ (or, since our treatment uses ordinals, $\underbar{\omega}$) the constant sequence always equal to 1. I mean: not introduce a new definition, but treat it on par with finite integers like in the "Remark" above.  Indeed, we want to "prove as much as possible of the resemblance of NNoo* and NNoo" to pinpoint the last pieces that evade constructionism.

Reply:
"Remark" makes sense at least for now. Like I said, I am having trouble coming up with compelling arguments one way or the other.